### ISSUE TITLE
[a11y] The `use saved card` button is way too close to the accessory view buttons

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/firefox-ios#15682]
## Steps to reproduce
1. Launch FF access checkout.stripe.com/dev
1. Tap on any field on the checkout page. 
1. Observe the `use saved card` and the accessory view buttons.
### Expected behavior
The buttons should have a space between them in order to make it easier for a11y.
### Actual behavior
The `use saved card` button is way too close to the accessory view buttons
### Device & build information

* Device: iPhone 13 Pro (15.7.1). 
* Build version: 116 (32820) 
* First seen version: N/a 
### Notes
Attachments: 

![image](https://github.com/mozilla-mobile/firefox-ios/assets/42831109/6af75c80-f891-4b21-bec6-0d17f4f3b2c6)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-7055)

**Comment by anna-yeddi** [comment]
More on touch target sizes on Android in [the BBC A11y Guide](https://www.bbc.co.uk/accessibility/forproducts/guides/mobile/target-touch-size/) and in [the WCAG 2.1 Success Criterion 2.5.5](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html).

But personally I think this would also improve usability for general audience as well.

**Comment by data-sync-user** [comment]
➤ Adina Petridean commented:

Verified as fixed on v9000(42096) with iPhone 14 Pro Max (iOS 16.2).

!UseSavedCard (1).jpeg|width=430,height=932,alt="UseSavedCard (1).jpeg"!

**Comment by data-sync-user** [comment]
➤ Alina Moldovan commented:

Verified as fixed using v128 (43182)
